libr/include/r_util/r_event.h

```diff
@@ -0,0 +1,33 @@
+/* radare - LGPL - Copyright 2018 - pancake */
+
+#ifndef R_EVENT_H
+#define R_EVENT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct r_event_t {
+	void *user;
```

Comment:
I think this is not necessary. Each event should have all the right data passed through `data`

Reply:
The idea is that REvent will be associated to RCore, this is the instance associated, this will be important to handle tasks and such. Mainly because who will send the event doesnt need to be n RCore, but the code that handles the events must be in RCore (as core plugins, whatever..)

Reply:
ok